README.md

```diff
@@ -1,14 +1,23 @@
 # PML analyzer
 
-The PML analyzer is an open source API providing a simple DSL to build 
-a description of the architecture of your chip based on the PHYLOG Model Language (PML). 
-From this representation a set of safety and interference model templates can be generated to perfom safety and 
+The PML analyzer is an open source API providing a simple DSL to build
+a description of the architecture of your chip based on the PHYLOG Model Language (PML).
+From this representation a set of safety and interference model templates can be generated to perfom safety and
```

Comment:
It may be worth thinking about a linter for developers of the project, to ensure styling and reformating are automated.

Reply:
You are right. You can find [https://docs.scala-lang.org/style/](here) the style formatting both for code and in-code documentation. Do you think this might do the trick? 
If it does, I will add it to the README.md, I should also add a contributing.md as you suggested wisely.